src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorTimer.kt

```diff
@@ -33,8 +33,8 @@ class GardenVisitorTimer {
     private val config get() = GardenAPI.config.visitors.timer
 
     private val timePattern by RepoPattern.pattern(
-        "garden.visitor.timer.time",
-        "§b§lVisitors: §r§f\\((?<time>.*)\\)"
+        "garden.visitor.timer.time.new",
```

Comment:
Sadly this needs to be a new pattern otherwise the user is spammed with number format exceptions 😔